packages/bezier-react/src/components/Text/Text.module.scss

```
@@ -37,68 +49,80 @@
     &.typo-11 {
       font-size: var(--typography-size-11-font-size);
       line-height: var(--typography-size-11-line-height);
+      --b-text-line-height: var(--typography-size-11-line-height);
```

```suggestion
      --b-text-font-size: var(--typography-size-11-font-size);
      --b-text-line-height: var(--typography-size-11-line-height);
```

```css
--b-text-color: inherit;
--b-text-line-clamp: 1;
--b-text-font-size: initial;
--b-text-line-height: initial;
--b-text-letter-spacing: initial;

.Text {
  font-size: var(--b-text-font-size);
  line-height: var(--b-text-line-height);
  letter-spacing: var(--b-text-letter-spacing);
  /* ... */
}
```

이렇게 하면 line-height의 중복을 줄일 수 있을 거 같아요.